src/hooks/useDisableScrolling.ts

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from 'react'
+import { isMobile } from 'utils/userAgent'
+
+export default function useDisableScrolling(disable: boolean | undefined | null) {
```

Comment:
Add a JSDoc, something like:

```suggestion
/** Disables scrolling on mobile when `true` is passed. */
export default function useDisableScrolling(disable: boolean | undefined | null) {
```